packages/core/src/controllers/relayer.ts

```diff
@@ -118,7 +125,8 @@ export class Relayer extends IRelayer {
     }
     this.initialized = true;
     setTimeout(async () => {
-      if (this.subscriber.topics.length === 0) {
+      // @ts-ignore
```

Comment:
```suggestion
      // @ts-expect-error - reason...
```

Reply:
Why does this `if` need a type override?

Reply:
subscriber.pending wasn't included in the types initially even though was public property